### ISSUE TITLE
Language abbreviation is incorrect ("gr" instead of "de")

### ISSUE THREAD

**Issue by DaBittna** [siropkin/kursor#10]
Hello,

since Kursor version 1.1.0 or 1.1.1, the German keyboard layout is displayed as "gr" (Greek?) instead of "de", as it was before.
![rider64_jGIc0bzkMY](https://github.com/user-attachments/assets/67b7e80f-23f5-4954-bf7a-9e2fdce449d4)
![mlr83unpn6](https://github.com/user-attachments/assets/40f0ae14-658c-4402-a3b7-6747f78b188b)

I'm using Rider 2024.1.4 on Windows 11.

**Comment by siropkin** [comment]
Thank you for the feedback! I used this [keyboard identifier list](https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/windows-language-pack-default-values?view=windows-11) and for some reason German is "GR" there... I fixed it to "DE" (and "DEI" for "German (IBM)" input type).

Fixed in Kursor v1.2.0 which now is in Jetbrains' review and will be available in marketplace in ~2days.
